gh-issues-import.py

```diff
@@ -404,6 +404,8 @@ def get_label_by_name(name):
 			del issue['label_objects']
 		
 		result_issue = send_request('target', "issues", issue)
+		if issue['state'] == "closed":
```

Comment:
an import failed for me with `KeyError: 'state'` for an issue — use `if issue.get('state') == "closed":` instead, this seems to work fine